API/controller/mealController.js

```diff
@@ -70,11 +70,26 @@ const updateAmeal = (req, res) => {
   });
 };
 
+// TO REMOVE A MEAL OPTION
+const deleteAmeal = (req, res) => {
+  const meal = dummydb.meals.find(data => data.id === parseInt(req.params.id));
+  if (!meal) res.status(404).send('meal with the given id was not found');
```

Comment:
You want to use `json()` instead of `send`, primarily because you aren't really sending anything when your server responds in this case. Also, with `json()` you can have key-value pairs that can be easy to make use of on the frontend. So the above status can be re-written as
```javascript
res.status(404).json({
  status: 'Not found',
  message: 'meal with the given id was not found'
})